modules/microshift-default-settings.adoc

```diff
@@ -46,21 +46,27 @@ ingress:
   routeAdmissionPolicy:
     namespaceOwnership: InterNamespaceAllowed # <12>
   status: Managed # <13>
-manifests: # <14>
+kubelet: # <14>
+manifests: # <15>
   kustomizePaths:
     - /usr/lib/microshift/manifests
     - /usr/lib/microshift/manifests.d/*
     - /etc/microshift/manifests
     - /etc/microshift/manifests.d/*
 network:
   clusterNetwork:
-    - 10.42.0.0/16 # <15>
+    - 10.42.0.0/16 # <16>
   serviceNetwork:
-    - 10.43.0.0/16 # <16>
-  serviceNodePortRange: 30000-32767 # <17>
+    - 10.43.0.0/16 # <17>
+  serviceNodePortRange: 30000-32767 # <18>
 node:
-  hostnameOverride: "" # <18>
-  nodeIP: "" # <19>
+  hostnameOverride: "" # <19>
+  nodeIP: "" # <20>
+  nodeIPv6: "" # <21>
+storage:
+    driver: "" # <22>
+    optionalCsiComponents:
+        - "" # <23>
```

Comment:
```suggestion
storage:
  driver: "" # <22>
  optionalCsiComponents:  # <23>
  - ""
```

Reply:
Moved the number since it's the `optionalCsiComponents` field that's actually being described below.